docs/requirements.txt

```diff
@@ -24,3 +24,4 @@ skorch==0.12.1
 torchvision==0.14.1
 torchaudio==0.13.1
 timm==0.6.12
+datasets==2.9.0
```

Comment:
```suggestion
datasets>=2.9.0
```

Reply:
I recommend just lower-bounding the version, because this will be easy way to assess that datalab still works with new versions of `datasets` package as they're released